pyEpiabm/pyEpiabm/intervention/household_quarantine.py

```diff
@@ -48,8 +49,17 @@ def __call__(self, time):
                         r_house = random.random()
                         if r_house < self.quarantine_house_compliant:
                             for household_person in person.household.persons:
-                                r_indiv = random.random()
-                                if r_indiv < self.\
-                                   quarantine_individual_compliant:
-                                    household_person.quarantine_start_time = \
-                                        time + self.quarantine_delay
+                                if not household_person.is_symptomatic():
```

Comment:
If there are multiple symptomatic individuals, will they all be case isolating?

Reply:
Yes, this is done in the 'case_isolation.py' script.